Typo "Pytorch"→"PyTorch"

hyoshioka0128 opened the pull request:
https://learn.microsoft.com/en-us/windows/ai/windows-ml/tutorials/pytorch-deploy-model 
https://github.com/MicrosoftDocs/windows-ai-docs/blob/docs/docs/windows-ml/tutorials/pytorch-deploy-model.md 
#PingMSFTDocs

opbld34 commented:
Docs Build status updates of commit _[a0ea23c](https://github.com/MicrosoftDocs/windows-ai-docs/pull/124/commits/a0ea23c22e9a12a9e8d08bd6eaa9f28b1c010942)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/vision-skills-diagram.png](https://github.com/MicrosoftDocs/windows-ai-docs/blob/user/lpbourret/updateWinVizSkillDiagram/docs/images/vision-skills-diagram.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/windows-vision-skills/index?branch=pr-en-us-124) |
[docs/images/vision-skills-diagram2.png](https://github.com/MicrosoftDocs/windows-ai-docs/blob/user/lpbourret/updateWinVizSkillDiagram/docs/images/vision-skills-diagram2.png) | :white_check_mark:Succeeded | |
[docs/windows-vision-skills/index.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/user/lpbourret/updateWinVizSkillDiagram/docs/windows-vision-skills/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/windows-vision-skills/index?branch=pr-en-us-124) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C18%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201904182123321833-124%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=2pWfd2%2B75YfZvpuFW%2FhJYcXhEhBT6a4%2BF2CP%2FIHR0J8%3D&st=2019-04-18T21%3A19%3A55Z&se=2019-05-19T21%3A24%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

hyoshioka0128 committed:
Typo "Pytorch"→"PyTorch"

https://learn.microsoft.com/en-us/windows/ai/windows-ml/tutorials/pytorch-deploy-model
https://github.com/MicrosoftDocs/windows-ai-docs/blob/docs/docs/windows-ml/tutorials/pytorch-deploy-model.md
#PingMSFTDocs
